packages/ember-glimmer/lib/utils/references.js

```diff
@@ -7,9 +7,13 @@ import { ConditionalReference as GlimmerConditionalReference, NULL_REFERENCE, UN
 import emberToBool from './to-bool';
 import { RECOMPUTE_TAG } from '../helper';
 import { dasherize } from 'ember-runtime/system/string';
+import { ARGS } from '../component';
 
 export const UPDATE = symbol('UPDATE');
-
+export const READONLY = symbol('READONLY');
+export const MUT = symbol('MUT');
+export const ACTION = symbol('ACTION');
+export const INVOKE = symbol('INVOKE');
 // FIXME: fix tests that uses a "fake" proxy (i.e. a POJOs that "happen" to
```

Comment:
Decoupling from htmlbars instead of importing the existing symbols. Let me know if this makes sense